tests/opentelemetry-docker-tests/tests/asyncpg/test_asyncpg_functional.py

```diff
@@ -14,7 +14,7 @@
 POSTGRES_USER = os.getenv("POSTGRESQL_HOST ", "testuser")
 
 
-def _await(coro):
+def async_call(coro):
```

Comment:
The same function exists in the aiopg tests under this name. I didn't really think the function was worth factoring out, but I thought it would be nice to at least name it consistently.